src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,126 @@
+package duke.ui;
+
+import duke.command.Command;
+import duke.command.AddCommand;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.exceptions.Exceptions;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+/**
+ * The Ui program gets user input, interprets it and calls relevant methods.
+ *
+ * @version 1.0
+ * @since 2020-01-28
+ */
+public class Ui {
+
+    Storage storage;
+    TaskList taskList;
+
+    /**
+     * Constructor.
+     *
+     * @param taskList refers to TaskList object required to manage the taskList.
+     */
+    public Ui(Storage storage, TaskList taskList) {
+
+        this.storage = storage;
+        this.taskList = taskList;
+
+    }
+
+    /**
+     * Takes in user input and calls relevant method.
+     */
+    public void frontDesk() {
+
+        final String HEADER = "____________________________________________________________\n";
```

Comment:
I've seen HEADER and FOOTER defined somewhere else... Maybe it could be better to have a class storing them as static variables?